node.js

```diff
@@ -13,12 +13,50 @@ module.exports = {
   },
 
   rules: {
-    'node/no-sync': [1],
+    // Best practices
     'node/global-require': [2],
     'node/handle-callback-err': [2],
+    'node/no-deprecated-api': [2],
+    'node/no-exports-assign': [2],
+    'node/no-extraneous-require': [2],
+    'node/no-missing-require': [2],
     'node/no-mixed-requires': [2],
     'node/no-new-require': [2],
     'node/no-path-concat': [2],
-  }
+    'node/no-process-env': [2],
+    'node/no-process-exit': [2],
+    'node/no-sync': [2],
+    'node/no-unsupported-features/es-builtins': [2],
+    'node/no-unsupported-features/es-syntax': [2],
+    'node/no-unsupported-features/node-builtins': [2],
```

Comment:
Won't these cause a lot of errors in projects without an `engines` field in the package.json, where the node version will default to v8.0.0?

Reply:
Yes. :)